### ISSUE TITLE
jdk_math_jre_0_FAILED Error: JDK not found

### ISSUE THREAD

**Issue by JasonFengJ9** [eclipse/openj9#7515]
Failure link
------------

https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_sanity.openjdk_ppc64le_linux_Release/1/consoleFull

Failure output (captured from console output)
---------------------------------------------

```
15:57:01  Error: JDK not found: /home/jenkins/jenkins-agent/workspace/Test_openjdk8_j9_sanity.openjdk_ppc64le_linux_Release/openjdkbinary/j2re-image
15:57:01  
15:57:01  jdk_math_jre_0_FAILED
```

**Comment by pshipton** [comment]
@smlambert mentioned: this test is there to verify that the jre builds at adopt ‘work’. But at OpenJ9 we don't archive the JRE build for jdk8, and I don't think we build it for jdk11+.

**Comment by pshipton** [comment]
@smlambert can we exclude this test in the OpenJ9 runs?

**Comment by smlambert** [comment]
We can exclude by implementation but not by CI server.  At the moment if I exclude it, it will be excluded at AdoptOpenJDK also.  

If you are building against nightly builds pulled from AdoptOpenJDK API, maybe should just additionally grab the JRE build at the same time.

Will think about ways to deal with it, other than the 2 mentioned above.

**Comment by pshipton** [comment]
I don't want to test against Adopt builds, but against the latest OpenJ9 build created at the OpenJ9 project.

**Comment by smlambert** [comment]
The short answer is that then OpenJ9 should archive the JRE that it builds, longer answer is we break that test out from the sanity set and run it as 'special' at Adopt, will need to think about this (and any complications it may introduce - fyi @sophia-guo).

**Comment by pshipton** [comment]
Archiving the JRE would consume more space, for something we don't really need. Moving stuff between the build machines and artifactory will take extra time, slowing down the builds. It's built for jdk8 but not sure it's already built for jdk11, adding it would slow down the builds a little.

**Comment by smlambert** [comment]
Yep, I am going to look at shifting it into special.openjdk target.  Need to make an update to build scripts at Adopt to add that top-level target and make sure that we still pick up jre from upstream build etc, but it will mean it will no longer be present in the sanity.openjdk target that you now want to run at the OpenJ9 project (and so will not fail as it won't get run here).
